### ISSUE TITLE
Tracker D - SMOD=3,0,1 (GPS + BLT)

### ISSUE THREAD

**Issue by kaciker** [dragino/TrackerD#27]
My Conf:
AT+MODEL=TrackerD ,v1.4.8

 
AT+ADR=1
AT+DR=5
AT+TXP=0
AT+NJM=1
AT+SMOD=3,0,1
AT+TDC=100000
AT+MTDC=300000
AT+ATDC=60000
AT+FTIME=180
AT+INTWK=0
AT+LON=1
AT+CHS=0
AT+CHE=1
AT+NMEA353=0
AT+NMEA886=0
AT+CFM=0
AT+PNACKMD=0
AT+PDOP=7.00
AT+DWELLT=0
AT+SHOWID=0
AT+BLEMASK=
AT+WiFiMASK=
AT+PT=14
AT+ATST=15
AT+PM=0
AT+FD=0
AT+BG=0
AT+BEEP=0
AT+EAT=2
AT+GF=1

The info in the Dragino doc.

3.2.7  Set Positioning Mode
SMOD define how TrackerD scan and uplink data:

AT Command:
AT+SMOD=aa,bb,cc

 aa:

1: GPS ONLY(Factory Settings):    Only get and uplink GPS location info.
2: BLE or WiFi ONLY:                     Only obtain iBeacon info via BLE and uplink  or obtain WiFi ssid info via WiFi and uplink. Design for Indoor tracking.
3: GPS/BLE Hybrid:                       Combination for Indoor and Outdoor tracking.Devices will try to search BLE iBeacon first. If device can't find the iBeacon, it will use GPS for positioning.
bb:

0 :  GPS+ BAT+ State+Tem&Hum
1 :  GPS +BAT State
cc:   

1 :  (iBeacon)UUID+ Major + Minor+Power+Rssi+BAT+State 
 
2 :  (WiFi)SSID+Rssi+BAT+State (V1.4.1 Version support this function later)   
Example:

  AT+SMOD=1,0,0    -->    GPS+ BAT+ State+Tem&Hum

  AT+SMOD=1,1,0    -->    GPS +BAT State

  AT+SMOD=2,0,1    -->    (iBeacon)UUID+ Major + Minor+Power+Rssi+BAT+State

  AT+SMOD=2,0,2    -->    (WiFi)SSID+Rssi+BAT+State 

  AT+SMOD=2,0,3    -->    (iBeacon) (Major + Minor+Rssi)+(Major + Minor+Rssi)+...(Maximum forty group)...+BAT+State

Downlink Payload (prefix 0xA5):
0xA5 01 00 00       //  Same as AT+SMOD=1,0,0

My problem and question:

Question. It is possible that there are some mistake in the doc ? the last example AT+SMOD=2,0,3 is not possilbe if we read the previous info due to in the AT+SMOD=aa,bb,cc  the cc has only two options iBeacon/wiifi, then , what about that last "3"  in the SMOD=2,0,3 ??

My problem.
I test all options with the aa in the SMOD=3,x,x and I never have ibeacon+gps I only can have ibeacon or gps, setting aa=1 or 2  but never both with option 3

**Comment by namejinnameyuan** [comment]
AT+SMOD=2,0,3, this is to obtain a lot of beacons, up to 40 are supported.
SMOD=3,x,x This is the hybrid mode. If no beacon is scanned, switch to GPS. This is to choose one of the two.

**Comment by kaciker** [comment]
> AT+SMOD=2,0,3, this is to obtain a lot of beacons, up to 40 are supported. SMOD=3,x,x This is the hybrid mode. If no beacon is scanned, switch to GPS. This is to choose one of the two.

You are totally right. The doc is clear.. my mistake, I was always setting the sensor with the BCN02 so close to it.. thanks, I have moved the devices away and started to report Fport2 with gps. 
![image](https://github.com/user-attachments/assets/12430012-8387-42a2-8d30-4dad03e3e1a7)


It is not possible to set the device to have temp/hum data, when  iBeacon is reporting ? doc inform that is only available in gps mode bb=0

**Comment by namejinnameyuan** [comment]
When uploading beacon data, temperature and humidity data are not added.
GPS adds temperature and humidity data, bb=0;
